ember-app/app/templates/milestones/create.hbs

```diff
@@ -20,7 +20,7 @@
         Due date: {{format-date dueDate}} 
         <small> <a href="#" {{action "clearDueDate"}}> clear </a> </small>
       </h5>
-      {{hb-date-picker dueDateBinding="model.due_on" processingBinding="processing"}}
+      {{hb-date-picker dueDate=model.due_on processing=processing }}
```

Comment:
notice they are unquoted

I think that may have been your issue, unquoted === bound property